tests/fixtures/nodes/src/nodes.rs

```diff
@@ -30,16 +30,16 @@ use std::{
     net::TcpStream,
     path::{Path, PathBuf},
     process::{Child, Command},
-    sync::atomic::{AtomicU64, Ordering},
+    sync::atomic::{AtomicU64, AtomicUsize, Ordering},
     thread::{self, sleep},
     time::{Duration, Instant},
 };
 use tempfile::TempDir;
-use tokio::runtime::Runtime;
+use tokio::{runtime::Runtime, sync::Mutex};
 use tracing_fixture::{tracing, Tracing};
 use xshell::{cmd, Shell};
 
-const PAYER_FUND_CHUNK: usize = 20;
+pub const MAX_PAYERS_NUM: usize = 2;
```

Comment:
The idea here was to fund lots of payers in a single TX to speed tests up. If this doesn't slow anything down we might as well have a single payer and get rid of all this pooling logic? e.g. if going from 40+ payers to 2 doesn't slow it down then making it go down to 1 won't either. 

I think this was added because of bors r+ tests though, so you likely won't see the slowdown locally. Did you see how much slower it is there?

Reply:
some test(s) failed when running all of them with 1 payer, i think some tests require two payers?, but let me double check that

Reply:
I forgot that we have `PrefundedAccount'`s which are used in tests. When prefunding is disabled, having more payers make tests run faster, 30 payers vs 1 => 9 sec vs 18 sec.
Also, I think for remote functional tests having more payers will also help, as tx commit times are much slower in testnet.